contracts/PoolManager.sol

```diff
@@ -132,55 +132,69 @@ contract PoolManager is IPoolManager, Owned, NoDelegateCall, ERC1155, IERC1155Re
     /// @inheritdoc IPoolManager
     mapping(Currency currency => uint256) public override reservesOf;
 
+    IPoolManager.Lock[] private locks;
+
     /// @inheritdoc IPoolManager
-    address[] public override lockedBy;
+    function locksGetter(uint256 i) external view override returns (IPoolManager.Lock memory) {
```

Comment:
so much for`extsload` 😉

Reply:
Why expose a getter and a length getter, and not just make it public?

Reply:
😛 yeah until storage is more or less frozen it's not really worth the trouble imo....

and yeah @hensha256 good point i made it public and removed the indexed getter, though you still need a manual getter for the length